### PR TITLE
Delete Qovery and Create ##Domain

### PR DESCRIPTION
**Pull request by DreamOfIce**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1031,6 +1030,14 @@ Table of Contents
 
 **[⬆ back to top](#table-of-contents)**
 
+## Domain
```
Comment (ripienaar) [inline README.md:1033]: Blank line below and please add to the top index

File: README.md [hunk 2]
```diff
@@ -996,7 +996,6 @@ Table of Contents
   * [txti.es](https://txti.es/) — Quickly create web pages with markdown.
   * [Vercel](https://vercel.com/) — Build, deploy, and host web apps with free SSL, global CDN, and unique Preview URLs each time you `git push`. Perfect for Next.js and other Static Site Generators.
   * [Versoly](https://versoly.com/) — SaaS focussed website builder - unlimited websites, 70+ blocks, 5 templates, custom CSS, favicon, SEO and forms. No custom domain.
-  * [Qovery](https://www.qovery.com) — Qovery is the simplest way to deploy your full-stack apps on AWS, GCP and Azure. It is free web hosting for developers with Database, SSL, a global CDN, and auto deploys from Git.
```
Comment (ripienaar) [inline README.md:999]: They do still appear to provide a free tier?

**Review by ripienaar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DreamOfIce) [reply README.md:1033]: OK

File: README.md (hunk 2 above)
Comment (DreamOfIce) [reply README.md:999]: In the past, its free program provided server settings. Later, they changed it to a community program. They need to continue to contribute to the qovery community in order to get free points See:  https://hub.qovery.com/docs/community/reward/feedback/
The current free plan needs to provide its own infrastructure. I don't think it belongs to as-a-service offers

Comment (ripienaar) [reply README.md:999]: Ah, I see once you click signup a bunch of b/s starts, yah lets remove them then.

File: README.md (hunk 1 above)
Comment (DreamOfIce) [reply README.md:1034]: I did

**Comment by ripienaar** [comment]
thanks

**Pull request merged**